src/tools/rbd_wnbd/rbd_wnbd.cc

```diff
@@ -1848,6 +1411,14 @@ static int rbd_wnbd(int argc, const char *argv[])
       if (construct_devpath_if_missing(&cfg)) {
         return -EINVAL;
       }
+      if (g_conf()->daemonize) {
+        r = send_map_request(get_cli_args());
```

Comment:
Could `cfg.command_line` be passed here instead of constructing the string from scratch?